roles/gpu_operator_bundle_from_commit/tasks/build_bundle.yml

```diff
@@ -63,7 +63,13 @@
   - nvidia-node-status-exporter
 
 - name: Apply the operator bundle image builder script manifest
-  command: oc apply -f "{{ gpu_operator_bundle_image_builder_script }}"
+  shell:
+    set -o pipefail;
+    oc create cm operator-bundle-image-builder-script
+       -n gpu-operator-ci
+       "--from-file=run_operator_bundle_image_builder.sh={{ gpu_operator_bundle_image_builder_script }}"
+       --dry-run -oyaml
```

Comment:
why dry run + | apply rather than just omitting the dry run flag? It would've been useful if you `tee`'d the configmap into the artifacts dir though, so maybe add that

Reply:
> why dry run + | apply

because 
- `oc create` doesn't update if exists (that's `oc apply`)
- `oc apply` doesn't support --from-file``

> `tee`'d the configmap into the artifacts dir

the CM is very static, you can deduce it from `name=operator-bundle-image-builder-script` `namespace=gpu-operator-ci` `data=run_operator_bundle_image_builder.sh={{ operator-bundle-image-builder-script }}`,
so I don't see any reason to store it in the artifacts dir
(I store the `ci-artifacts` HEAD [commit](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/logs/periodic-ci-openshift-psap-ci-artifacts-release-4.9-gpu-operator-e2e-master/1462556576953929728/artifacts/gpu-operator-e2e-master/nightly/artifacts/ci_artifact.git_version) in the directory, so that you can reconstruct the exact content of the script file, if ever you need)

Reply:
Ah that makes sense. I thought you just wanted to `tee` and forgot